src/packages/cli/src/index.ts

```diff
@@ -20,7 +20,7 @@ import { config } from '@exogee/graphweaver-config';
 
 const MINIMUM_NODE_SUPPORTED = '18.0.0';
 
-yargs
+yargs()
```

Comment:
Keep track of the new yargs instance you are creating here, say:

```suggestion
const yargsInstance = yargs()
```